### PR TITLE
GO-3127: Leave request approve method

### PR DESCRIPTION
**Pull request by mcrakhman**
Add SpaceLeaveApprove request:
- check that we do have a remove request from such participants
- if yes delete them
- if no send an error

### CODE REVIEW

File: core/acl/aclservice.go [hunk 1]
```diff
@@ -195,6 +197,37 @@ func (a *aclService) ChangePermissions(ctx context.Context, spaceId string, perm
...
+			if rec.RequestIdentity.Equals(identity) {
+				delete(identitiesMap, identity.Account())
+			}
+		}
+	}
+	if len(identitiesMap) != 0 {
+		acl.RUnlock()
+		var identitiesString string
+		for identity := range identitiesMap {
+			identitiesString += identity + ", "
```
Comment (fat-fellow) [inline core/acl/aclservice.go:223]: Stringbuffer?

Comment (mcrakhman) [reply core/acl/aclservice.go:223]: This is just an error to print the identities :-) It doesn't matter

Comment (mcrakhman) [reply core/acl/aclservice.go:223]: But it is better to just do strings with separator though

Comment (mcrakhman) [reply core/acl/aclservice.go]: Changed that

**Pull request merged**